control-plane/agents/src/bin/core/controller/policies/rebuild_policies.rs

```diff
@@ -16,11 +16,12 @@ use std::time::Duration;
 */
 
 // time constants, in seconds
-const TWAIT_SPERF: std::time::Duration = Duration::new(600, 0);
-const TWAIT_SAVAIL: std::time::Duration = Duration::new(300, 0);
-const TWAIT_ZERO: std::time::Duration = Duration::new(0, 0);
+pub const TWAIT_SPERF: std::time::Duration = Duration::new(600, 0);
+pub const TWAIT_SAVAIL: std::time::Duration = Duration::new(300, 0);
+pub const TWAIT_ZERO: std::time::Duration = Duration::new(0, 0);
```

Comment:
what do these mean, the name is not very descriptive, at the very least need some comments.